pageserver/src/page_cache.rs

```diff
@@ -33,25 +58,27 @@ pub fn get() -> Option<&'static PageCache> {
 const PAGE_SZ: usize = postgres_ffi::pg_constants::BLCKSZ as usize;
 const MAX_USAGE_COUNT: u8 = 5;
 
+///
+/// MapKey uniquely identifies a "thing" to cache in the page cache.
+///
+#[derive(PartialEq, Eq, Clone)]
+enum MapKey {
```

Comment:
Since this isn't for the map anymore, maybe we should just call it `Key` or `CacheKey`